config/initializers/rate_limiter.rb

```diff
@@ -51,9 +44,9 @@ class Rack::Attack
       end
     end
 
-    throttle("throttle_user_activate", RateLimit.user_api_options) do |req|
+    throttle("throttle_user_activate", RateLimit.auth_api_options) do |req|
       if req.post? && req.path.start_with?("/api/v4/users/activate") && SimpleServer.env.production?
-        req.ip
+        req.ip && req.params['user']['id']
```

Comment:
This is generating a boolean result, right? I believe that's the wrong thing to return here. We need to return a unique "cache key" that will be used to categorize each rate limit bucket. I think we've accidentally created only 2 buckets - true and false. We probably want to do something like 

`"#{req.ip} #{req.params['user'][id]}`

Reply:
Ah, additionally, I think our specs will need to be updated so that in addition to `expect to be blocked when i > limit`, we also need the `expect _not_ to be blocked when i <= limit`. That would have surfaced this problem.

Reply:
> This is generating a boolean result, right?

Since the IP would always exist, this would actually just return the user-id. Which is, accidentally, correct 😁 . We do have the other test as well, actually.

I've changed this in ae2174f1566369613b1b87fa9b5eb56da68aaa9a to return just the user-id. This should be alright now, ya?

Reply:
Ha, that's amazing 😂 Yeah just a user ID should be great, and is simple to grok as well.